md-rest-api/src/main/java/com/wipro/ats/bdre/md/rest/ProcessAPI.java

```diff
@@ -371,7 +375,7 @@ RestWrapper export(HttpServletResponse resp,
                        @PathVariable("id") Integer processId
     ) {
         RestWrapper restWrapper = null;
```

Comment:
undo changes made in export function, you have commented resp.setHeader("Content-Disposition", "attachment; filename=" + processId + ".json");